tests/evm/test_oog_constant.py

```diff
@@ -60,7 +60,7 @@ def test_oog_constant_root(tx: Transaction, bytecode: Bytecode):
                 program_counter=0,
                 stack_pointer=1023,
                 gas_left=2,
-                reversible_write_counter=2,
+                reversible_write_counter=0,
```

Comment:
Why not update the next step to have `rw_counter=28`? I don't think any step will have `reversible_write_counter=0`

Reply:
sound good & updated in [4879fb5](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/280/commits/4879fb507b64b8da492d85b52508114196e5641d) thanks!